framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -142,7 +149,7 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 		// Fetch last certificate from the receiving chain and update the _lastCertificate
 		const chainAccountJSON = await this._receivingChainClient.invoke<ChainAccountJSON>(
 			'interoperability_getChainAccount',
-			{ chainID: this._ownChainID },
+			{ chainID: this._ownChainID.toString('hex') },
```

Comment:
- first 2 points done
- since we are not reusing this code we can skip it as of now. I have an idea for refactoring later, in the sense that we can have a class SendingChainClient and ReceivingChainClient where we can encapsulate certain functions and simply use them like,
`receivingChainClient.getLastCertificate()`
`sendingChainClient.getOwnAccount()`

Reply:
created related issue https://github.com/LiskHQ/lisk-sdk/issues/8175